compiler/rustc_codegen_ssa/src/sir.rs

```diff
@@ -890,3 +890,136 @@ impl SirTypes {
         })
     }
 }
+
+pub mod labels {
+    use object::{Object, ObjectSection};
+    use std::{
+        convert::TryFrom,
+        fs,
+        path::{Path, PathBuf},
+        process::Command,
+    };
+
+    fn split_symbol(s: &str) -> (String, u32) {
```

Comment:
Is `u32` forced upon us or have we chosen it?

Reply:
We've inherited that from MIR. We could change it later, but there's no need at this point in time since our BB indices are a one-to-one mapping with those of MIR (at the moment).

Reply:
Fair enough.